examples/websocket.go

```diff
@@ -22,8 +22,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// Websocket strategy
-func Websocket() strategies.Strategy {
+var (
+	Websocket = websocket()
```

Comment:
why using this if it's not called twice? just leave original assignment as it was before
``` go
strategies.WebsocketStrategy{
		Model: strategies.StrategyModel{
			Name: "Websocket",
			Setup: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				for _, wrapper := range wrappers {
					err := wrapper.FeedConnect(markets)
					if err == exchanges.ErrWebsocketNotSupported || err == nil {
						continue
					}
					return err
				}
				return nil
			},
			OnUpdate: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				// do something
				return nil
			},
			TearDown: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				return nil
			},
			OnError: func(err error) {
				logrus.Error(err)
			},
		},
	}
```

Reply:
I created all variables calling to their functions
```
var (
	Watch5Sec                  = watch5Sec()
	SlackIntegrationExample    = slackIntegrationExample()
	TelegramIntegrationExample = telegramIntegrationExample()
)
``` 
and 
```
var (
	Websocket = websocket()
)
```
Just trying to be tidy

Reply:
Maybe should be `WebsocketExample = websocketExample()` and the same with `watch5Sec`

Reply:
I dont want that, since we refer only to the global variable and use functions only once